common/src/test/java/org/opensearch/ml/common/model/GuardrailsTests.java

```diff
@@ -83,4 +83,24 @@ public void parse() throws IOException {
         Assert.assertEquals(guardrails.getInputGuardrail(), inputLocalRegexGuardrail);
         Assert.assertEquals(guardrails.getOutputGuardrail(), outputLocalRegexGuardrail);
     }
+
```

Comment:
Can we add a test where the type is model, it fetches the type `model` correctly.

Reply:
Added.